charts/rancher-logging/0.4.0/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 description: Rancher logging helm chart to support logging function in rancher
 icon: https://raw.githubusercontent.com/fluent/fluentd-docs/master/public/logo/Fluentd_square.png
 name: rancher-logging
-version: 0.3.0
+version: 0.4.0
```

Comment:
Note: I fixed the version which was missed to be updated to 0.4.0 when the chart was introduced.